### PR TITLE
remove 'parl' directory and add a quick tutorial

### PR DESCRIPTION
**Pull request by yu239**
We need an IO page for a comprehensive tutorial in the future.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,5 +20,97 @@ FLARE is currently a pure Python framework which doesn't need build. Several dep
...
+## For this simple problem, our model uses an MLP to directly map
+## each input vector to a latent embedding of 128 dimensions.
+mlp = nn.Sequential(
+    nn.Linear(state_shape, 128),
+    nn.ReLU(),
+    nn.Linear(128, 128), nn.ReLU(), nn.Linear(128, 128), nn.ReLU())
+
+## SimpleModelAC will be responsible for generating a discrete policy and a value
+## function from the generated latent embedding.
+## We use simple Actor-critic to train the model.
```
Comment (skylian) [inline README.md]: The learning algorithm is specified through defining an `Algorithm` object, which takes a `Model` object as its argument. In this example, we uses a pre-defined algorithm`SimpleAC`; it implements a simple Actor-critic algorithm, which takes the latent embedding output by the model defined above to generate a discrete policy and a value.

File: README.md [hunk 2]
```diff
@@ -20,5 +20,97 @@ FLARE is currently a pure Python framework which doesn't need build. Several dep
...
+
+3. Next we construct the neural network and choose the algorithm to train the network.
+
+```python
+## For this simple problem, our model uses an MLP to directly map
+## each input vector to a latent embedding of 128 dimensions.
+mlp = nn.Sequential(
+    nn.Linear(state_shape, 128),
+    nn.ReLU(),
+    nn.Linear(128, 128), nn.ReLU(), nn.Linear(128, 128), nn.ReLU())
```
Comment (skylian) [inline README.md]: `model = model=SimpleModelAC(dims=state_shape, num_actions=num_actions, mlp=mlp)`

File: README.md [hunk 3]
```diff
@@ -20,5 +20,97 @@ FLARE is currently a pure Python framework which doesn't need build. Several dep
...
+
+## SimpleModelAC will be responsible for generating a discrete policy and a value
+## function from the generated latent embedding.
+## We use simple Actor-critic to train the model.
+alg = SimpleAC(model=SimpleModelAC(
+    dims=state_shape, num_actions=num_actions, mlp=mlp))
```
Comment (skylian) [inline README.md]: `alg = SimpleAC(model=model)

File: README.md [hunk 4]
```diff
@@ -20,5 +20,97 @@ FLARE is currently a pure Python framework which doesn't need build. Several dep
...
+## get the environment observation dimension and the number of actions
+state_shape = envs[-1].observation_space.shape[0]
+num_actions = envs[-1].action_space.n
+```
+
+3. Next we construct the neural network and choose the algorithm to train the network.
+
+```python
+## For this simple problem, our model uses an MLP to directly map
+## each input vector to a latent embedding of 128 dimensions.
```
Comment (skylian) [inline README.md]: We first create a `Model` object, `SimpleModelAC`, and define the network structure for it. For this simple problem, our model uses an MLP to directly map each input vector to a latent embedding of 128 dimensions.

File: README.md [hunk 5]
```diff
@@ -20,5 +21,111 @@ FLARE is currently a pure Python framework which doesn't need build. Several dep
...
+    manager.add_agent(agent)
+
+manager.start()
+```
+
+Suppose that the root directory of FLARE is `<flare_root>`. To run the above code, you can simply do
+```bash
+cd <flare_root>;
+python flare/examples/ac_example.py
+```
```
Comment (Haichao-Zhang) [inline README.md]: It would be useful to add a sentence describing what will be expected in the next few minutes after kicking off training (e.g. ~200 average reward in ~X minutes on a machine with Y GPU).

Comment (yu239) [reply README.md]: added
